src/ui/style/palette.scss

```diff
@@ -1,193 +1,101 @@
-$dropdown-background-dark: #242433;
-$dropdown-border-light: rgba(0, 64, 115, 0.12);
-$dropdown-tags-color: $primary;
-$dropdown-tags-background: rgba(0, 103, 150, 0.12);
-$dropdown-content-background-light: #ffffff;
-$dropdown-content-background-dark: $dropdown-background-dark;
-$dropdown-content-box-shadow-light: 0 4px 8px 0 rgba(0, 0, 0, 0.14);
-$dropdown-content-box-shadow-dark: 0 8px 20px 0 rgba(0, 0, 0, 0.40);
-
-
-// modal
-$modal-background-dark: linear-gradient(360deg, #242433 0%, #2f2f42 100%);
-$modal-background-light: linear-gradient(360deg, #f2f6f9 0%, #ffffff 100%);
-$modal-line-divider-light: rgba(0, 64, 115, 0.12);
-$modal-line-divider-dark: rgba(255, 255, 255, 0.12);
-// share dataset summary
-$share-dataset-summary-title-dark: rgba(255, 255, 255, 0.4);
-$share-dataset-summary-title-light: rgba(0, 64, 115, 0.4);
-
-// daypicker
-$daypicker-background-light: #ffffff;
-$daypicker-background-dark: #242433;
-$daypicker-day-color-light: $neutral-variant-2;
-$daypicker-day-color-dark: #E3E3E3;
-$daypicker-day-outside-color-light: $neutral;
-$daypicker-day-outside-color-dark: #707275;
-$daypicker-day-range-middle-light: $primary-variant-6;
-$daypicker-day-range-middle-dark: #203F5A;
-$daypicker-day-today-light: $primary-variant-1;
-$daypicker-day-today-dark: #2B608A;
-$daypicker-dropdowns-background-light: rgba($primary-variant-1, 0.08);
-$daypicker-dropdowns-background-dark: $background-dark;
+// Background
+$background-color-inverse: rgb(23, 24, 33);
+$background-color-dark: rgb(18, 19, 32);
+$background-color-inverse-light: rgb(36, 36, 51);
+$background-color: rgb(223, 228, 233);
+$background-color-light: rgb(242, 246, 249);
+
+// Blue Palette
+$blue-100: rgb(170, 204, 220);
+$blue-200: rgb(128, 179, 202);
+$blue-300: rgb(85, 154, 185);
+$blue-400: rgb(43, 128, 167);
+$blue-500: rgb(0, 103, 150);
+$blue-600: rgb(0, 86, 125);
+$blue-700: rgb(0, 69, 100);
+$blue-800: rgb(0, 52, 75);
+$blue-900: rgb(0, 48, 58);
+
+// Orange Palette
+$orange-100: rgb(255, 220, 170);
+$orange-200: rgb(255, 202, 128);
+$orange-300: rgb(255, 184, 85);
+$orange-400: rgb(255, 167, 43);
+$orange-500: rgb(255, 149, 0);
+$orange-600: rgb(212, 124, 0);
+$orange-700: rgb(170, 99, 0);
+$orange-800: rgb(128, 75, 0);
+$orange-900: rgb(85, 50, 0);
+
+// Neutral Palette
+$neutral-100: rgb(240, 243, 246);
+$neutral-200: rgb(229, 235, 240);
+$neutral-300: rgb(216, 225, 232);
+$neutral-400: rgb(204, 215, 224);
+$neutral-500: rgb(191, 205, 217);
+$neutral-600: rgb(178, 195, 209);
+$neutral-700: rgb(148, 162, 174);
+$neutral-800: rgb(119, 130, 139);
+$neutral-900: rgb(89, 98, 105);
+
+// Dark blue Palette
+$darkblue-100: rgb(170, 191, 208);
+$darkblue-200: rgb(128, 159, 185);
+$darkblue-300: rgb(85, 128, 162);
+$darkblue-400: rgb(43, 96, 138);
+$darkblue-500: rgb(0, 64, 115);
+$darkblue-600: rgb(0, 53, 96);
+$darkblue-700: rgb(0, 43, 77);
+$darkblue-800: rgb(0, 32, 58);
+$darkblue-900: rgb(0, 21, 38);
+$darkblue-100: rgb(170, 191, 208);
```

Comment:
`darkblue-100` is a duplicate ?

Reply:
Good catch! duplicated variable has been removed. Thanks